packages/react/src/components/ListBox/ListBoxMenuIcon.js

```diff
@@ -28,13 +28,12 @@ const defaultTranslations = {
  * state of the menu for a given `ListBox`
  */
 const ListBoxMenuIcon = ({ isOpen, translateWithId: t }) => {
-  const className = cx({
-    [`${prefix}--list-box__menu-icon`]: true,
+  const className = cx(`${prefix}--list-box__menu-icon`, {
```

Comment:
Any reason for the style change? Might help to keep calls to `cx` monomorphic if we can

Reply:
Looks that this change simply distinguishes constant styles vs. conditional styles.

Reply:
Not sure I understand, these are equivalent ways of expressing the same output. I left the note of monomorhpism because `classnames` supports variadic args. If we can defer to one style then it can be optimized in v8 (albeit by an arbitrary amount for this code path). If it's a code style question, then we should bring it up and discuss.

Reply:
In case you are thinking of V8 optimization with one style, would you elaborate that? Thanks!